modules/vba_documents/lib/vba_documents/pdf_inspector.rb

```diff
@@ -74,8 +74,8 @@ def inspect_pdf(add_file_key)
 
     def add_line_of_business(data, parts_metadata)
       if parts_metadata.key? 'businessLine'
-        data['line_of_business'] = parts_metadata['businessLine']
-        data['submitted_line_of_business'] = VALID_LOB[parts_metadata['businessLine']].to_s
+        data['line_of_business'] = parts_metadata['businessLine'].to_s.upcase
+        data['submitted_line_of_business'] = VALID_LOB[parts_metadata['businessLine'].to_s.upcase]
```

Comment:
I see that the to_s was on the outside of the closing bracket and now moved inside. Just checking to make sure this is what's expected

Reply:
Yes that is what is expected. VALID_LOB has string keys, and one requirement was to capitalize businessLine for the users. If a user submits a nonstring, we cannot capitalize it unless we first convert to a string. This way we will throw errors for nonstring values without having to directly check them because they won't be in the VALID_LOB keys